### PR TITLE
Databc api

### PR DESCRIPTION
**Pull request by popkinj**
API end point for fetching contextual data for DataBC.

### CODE REVIEW

File: api-mobile/src/paths/context/databc/{wfs}.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    },
+    default: {
+      $ref: '#/components/responses/default'
+    }
+  }
+};
+
+
+/**
+ * Fetches all activity records based on request search filter criteria.
```
Comment (micheal-w-wells) [inline api-mobile/src/paths/context/databc/{wfs}.ts]: I'm guilty of this too, but can you update this comment?

**Review by micheal-w-wells — DISMISSED** [review]
LGTMY other than the one comment

File: api-mobile/src/paths/context/databc/{wfs}.ts (hunk 1 above)
Comment (popkinj) [reply api-mobile/src/paths/context/databc/{wfs}.ts]: Ah yes... Nice catch @micheal-w-wells .

Comment (popkinj) [reply api-mobile/src/paths/context/databc/{wfs}.ts]: Also found some incorrect stuff in the OpenAPI def.

**Comment by popkinj** [comment]
Hey @micheal-w-wells can you have a quick look-see at this. Approve if happy.
Thanks

**Review by micheal-w-wells — APPROVED** [review]

**Comment by micheal-w-wells** [comment]
@popkinj good to go.  mind returning favour on https://github.com/bcgov/invasivesbc/pull/150 ? also let me know when you have time to link up on leaflet stuff, can't ping you on teams

**Comment by popkinj** [comment]
Thanks @micheal-w-wells ... Yeah, I'll have a look.
I have a short meeting at 1:30. Do you want to chat after that?

**Pull request merged**